### PR TITLE
Re-add 2 Fedora hard dependencies (rebased)

### PR DESCRIPTION
**Pull request by linux-modder**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ The following steps assumes that you cloned the repository at "/home/user/qTox".
 
 
 <a name="other-deps" />
-### GCC, Qt, FFmpeg, OpanAL Soft and qrencode
+### GCC, Qt, FFmpeg, OpanAL Soft and qrencode ffmpeg-devel
```
Comment (zetok) [inline INSTALL.md]: this change isn't needed

File: INSTALL.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ sudo apt-get install build-essential qt5-qmake qt5-default qttools5-dev-tools li
 #### Fedora:
 ```bash
 sudo dnf group install "Development Tools"
-sudo dnf install qt-devel qt-doc qt-creator qt5-qtsvg qt5-qtsvg-devel openal-soft-devel libXScrnSaver-devel qrencode-devel
+sudo dnf install qt-devel qt-doc qt-creator qt5-qtsvg qt5-qtsvg-devel openal-soft-devel libXScrnSaver-devel qrencode-devel qtsingleapplication  ffmpeg-devel
```
Comment (zetok) [inline INSTALL.md]: could you use 1 space, instead of 2?

**Comment by linux-modder** [comment]
@zetok, that look more  like it  should now?

**Comment by zetok** [comment]
@linux-modder yes. Could you made that change in 1 commit (current PR has already 2)

**Comment by linux-modder** [comment]
in 1 commit?   though I did that but  sure


Corey W Sheldon
Freelance IT Consultant, Multi-Discipline Tutor
Ameridea LLC, Co-Founder, CTO
Fedora Ambassador, North America
(p) +1 (310) 909-7672

    "Have no way as way, no limitation as limitation. One must never
underestimate the power of boredom...from which creativity and laziness are
borne, which can spark great works of chaos and genius."

Find Me  on any of the sites  I teach /frequent:
https://gist.github.com/linux-modder/ac5dc6fa211315c633c9
---------------------------------------------------------------------------------------------------
Tox:
corey84@toxme.se
9357BC6A5944A08AFC7D1EFFD61F6A73B9EABF8B2FB84ACF1DAC9A1A4D0A4705FFCCD0E5499B
PGP: 718BF597
<http://pgp.mit.edu/pks/lookup?op=get&search=0xE958C5D6718BF597> FP: 2930
99EB 083D D332 0752 88C4 E958 C5D6 718B F597
---------------------------------------------------------------------------------------------------

**Comment by linux-modder** [comment]
Thought I had.  sure can

**Pull request closed without merging**